### DISCUSSION TITLE
librivox

### DISCUSSION THREAD

**Discussion by ghost** [GrakovNe/lissen-android#34] [General]
The librivox Audiobook are stored at 
https://archive.org/details/librivoxaudio
https://librivox.org
https://github.com/LibriVox

---
---
If it is possible to transfer the lissen-android Login moved to:  Bookshelf>Settings>login

**Comment by GrakovNe** [comment]
Hi, thanks for the discussion!

First, I want to clarify that the Lissen app is not related to Librivox and is not a Librivox client app, but thanks for the links.

Second, there are no special login settings in the app. To log into the ABS server, you simply enter your credentials on the login screen.

After that, you can change your credentials by clicking the trash icon in settings. This action clears the app's authentication in ABS and redirects you back to the login page.

So, I can't "move" login settings because they don't exist.
